gemd/units/citrine_en.txt

```diff
@@ -218,7 +219,10 @@ week = 7 * day = Week
 fortnight = 2 * week = Fortnight
 year = 365.25 * day = yr = julian_year = Year = Julian_year = Julian_Year
 month = year / 12 = _ = Month
-decade = 10 * year = _ = Decade
+
+# decade = 10 * year = _ = Decade
```

Comment:
Why is this left commented?

Reply:
Again, this is in the upstream file, plus a low probability that it's domain relevant
https://github.com/hgrecco/pint/blob/f334aa8d1e754f1a0304a7416117a9a77591bd9a/pint/default_en.txt#L179-L180